qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDatabaseQueries.groovy

```diff
@@ -358,12 +359,20 @@ class CustomerDatabaseQueries {
             try {
                 createNewAffiliation(it, affiliation)
                 connection.commit()
-            } catch (Exception e) {
+            }
+            catch(DatabaseQueryException e) {
+                log.error(e.message)
+                log.error(e.stackTrace.join("\n"))
+                connection.rollback()
+                connection.close()
+                throw new DatabaseQueryException("Could not create affiliation in database")
```

Comment:
I meant `throw new DatabaseQueryException(e.message)`

and for exceptions still the "Could not create affiliation"